src/microbe_stage/editor/PatchMapDrawer.cs

```diff
@@ -1109,6 +1126,58 @@ private void AddPatchNode(Patch patch, Vector2 position)
 
         patch.ApplyPatchEventVisuals(node);
 
+        var playerSpecies = patch.FindSpeciesByID(PlayerSpeciesID);
+        if (playerSpecies != null)
+        {
+            var playerPopulationIndicatorAmount = (int)Math.Ceiling(
+                patch.GetSpeciesSimulationPopulation(playerSpecies) * 0.004);
```

Comment:
Making the constant here into a value in Constants would make it easier for someone else to tweak how many circles appear.